packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.ts

```diff
@@ -25,11 +25,11 @@ export const appBadges = `
   <mj-section>
     <mj-column>
       <% if (locals.onDesktopOrTabletDevice) { %>
-      <mj-text css-class="secondary-text mb-3"><span data-l10n-id="another-desktop-device"> Or, install on
+      <mj-text css-class="secondary-text mb-5"><span data-l10n-id="another-desktop-device"> Or, install on
```

Comment:
Ooh @Yogita98 I think the wrong class was changed. I think you can actually remove this `mb-5` from this text el, this is what I'm seeing right now:

<img width="958" alt="image" src="https://user-images.githubusercontent.com/13018240/129397899-94f3fd2a-952b-4fb6-88ba-39513e081278.png">

The `margin` between those two pieces of text is coming from the text below it:
<img width="535" alt="image" src="https://user-images.githubusercontent.com/13018240/129397955-ee7bb694-29c0-4c06-84ae-b9139317f81a.png">


That spacing is fine (the class can be removed and there will be no visual change), but what I'm thinking we need to change is `mb-2` on app badges:

<img width="915" alt="image" src="https://user-images.githubusercontent.com/13018240/129398098-73e77a3c-bde6-4dcf-adf1-797bbe6c9ea2.png">

Which looks like needs changing in `emails/partials/appBadges/index.scss` from `global.$s-3` to `global.$s-5`.

Reply:
Ahhh yes. Even I couldn't notice any visual changes when I kept the versions parallel to each other, thought something's wrong with my eyes lol. Btw are you using any tool while inspecting HTML, those dotted lines and colored spacing looks 🔥

Reply:
When there's true adjacent elements, [the top/bottom margin actually gets collapsed](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) to whatever value is larger. Padding doesn't do that for what it's worth. 🙂

I'm actually just using Firefox Nightly! 😁 Firefox Developer Edition has some cool looking tools too though, sometimes I'll use that.